extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/connection/ModeledConnection.java

```diff
@@ -157,6 +158,12 @@ public void setConfiguration(GuacamoleConfiguration config) {
     }
 
     @Override
+    public Set<String> getSharingProfileIdentifiers()
```

Comment:
So I don't see any implementation "getSharingProfileIdentifiers" that actually returns anything - am I missing something about how this works?